### PR TITLE
Update de_de.json

### PR DESCRIPTION
**Pull request by RazzorX**

### CODE REVIEW

**Comment by RazzorX** [comment]
German translation should be complete.
I hope I haven't forgotten anything...

gui.jade.translators
The previous translators should still be taken into account here! (currently missing)

What does the entry ' *_extra_msg ' stand for? Should these entries also be translated?

**Comment by Snownee** [comment]
I think it has been described here: https://github.com/Snownee/Jade/tree/1.21-fabric/src/main/resources/assets/jade/lang

File: src/main/resources/assets/jade/lang/de_de.json [hunk 1]
```diff
@@ -92,32 +202,114 @@
...
+	"jade.theme.jade.create": "Erstellen",
+	"jade.collectingItems": "Sammle Gegenstände...",
+	"config.jade.plugin_jade.block_properties": "Block-Eigenschaften",
+	"jade.block_explosion_resistance": "Explosionswiderstand: %s",
+	"jade.block_destroy_time": "Zerstörungszeit: %s",
+	"jade.block_jump_factor": "Sprungfaktor: %s",
+	"jade.block_speed_factor": "Geschwindigkeitsfaktor: %s",
+	"jade.block_ignite_odds": "Entzündungs-Chance: %s",
+	"jade.block_burn_odds": "Verbrennungs-Chance: %s",
+	"jade.ignore_list.comment": "Dies ist eine Ignorierliste für das Ziel von Jade. Du kannst Registrierungs-IDs zu der \"Werte\"-Liste hinzufügen.",
```
Comment (Snownee) [inline src/main/resources/assets/jade/lang/de_de.json:308]: the quoted word "values" should not be translated

File: src/main/resources/assets/jade/lang/de_de.json [hunk 2]
```diff
@@ -1,81 +1,191 @@
...
-	"tooltip.jade.state_on": "Ein",
+	"tooltip.jade.state_on": "An",
 	"tooltip.jade.state_off": "Aus",
 	"tooltip.jade.power": "Stärke: %d",
+	"jade.input_signal": "Eingang: %d",
 	"tooltip.jade.villager_profession": "Beruf: %s",
+	"gui.jade.by": "Erstellt mit %s von Snownee",
+	"gui.jade.by.hovered": "Sieh dir alle meine Arbeiten an!",
+	"gui.jade.translated_by": "Deutsche Übersetzung von %s",
+	"gui.jade.translators": "RazzorX",
```
Comment (Snownee) [inline src/main/resources/assets/jade/lang/de_de.json:20]: please add previous translators

**Review by Snownee — CHANGES_REQUESTED** [review]

**Pull request closed without merging**